### PR TITLE
Fix some bits in the readme

### PR DESCRIPTION
**Pull request by edwilde**
...and it isn't even hacktoberfest yet 😁 

* Fixed the duplicate on line 34 which was breaking the code block
* ~Upgraded the `**Note**` syntax with the proper [Github alert syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,9 +230,9 @@ To customise existing block templates such as `Content` and `Form` templates, co
 `vendor/dnadesign/silverstripe-elemental/templates` to your theme. When doing this, ensure you match the folder
 structure (PHP class namespace) to ensure that your new template version takes priority.
 
-**Note:** The default set of elements follow the [BEM (Block Element Modifier])(http://getbem.com/) class naming
-convention, which allows developers to style individual parts of the DOM without unnecessarily nested CSS. Where
-possible, we encourage you to follow this naming system.
+> [!NOTE]
+> The default set of elements follow the [BEM (Block Element Modifier])(http://getbem.com/) class naming convention, which allows developers to style individual parts of the DOM without unnecessarily nested CSS.
+> Where possible, we encourage you to follow this naming system.
```
Comment (GuySartorelli) [inline README.md]: Can you please revert this formatting change? Ideally we don't want any github-specific markdown, as that reduces the portability of the documentation and it is very likely that this documentation will be ported to the main developer-docs repo in the future.

Same with any other similar changes in this PR.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Thanks for contributing this. Just a small change requested below.

File: README.md (hunk 1 above)
Comment (edwilde) [reply README.md]: Ah, k..no worries

**Review by GuySartorelli — APPROVED** [review]
Nice one, thanks! I'll merge on green

**Pull request merged**